lawlibrary/templates/lawlibrary/court_detail.html

```diff
@@ -0,0 +1,46 @@
+{% extends 'peachjam/layouts/document_list.html' %}
+{% load i18n %}
+
+{% block title %}{% trans 'Judgement List' %}{% endblock %}
+
+{% block page-title %}
+    <h1 class="m-0">{% trans 'Judgements List' %}</h1>
```

Comment:
Rather use the court (author) name; if there's a year, append it.

eg. "Supreme Court" and "Supreme Court - 2021"

Reply:
done